### ISSUE TITLE
Why does it place thin hairs further into the background?

### ISSUE THREAD

**Issue by noobtoob4lyfe** [apple/ml-depth-pro#52]
In your example image of the camel, you can see that it's pushing the thinner hairs further into the background when I would expect the them to be at the same depth as the camel's head.  Is there a way to control or correct this?  

![image](https://github.com/user-attachments/assets/9fb90967-b47c-4f73-a172-d98df2ae08bb)

**Comment by Keivin98** [comment]
+1
Were you able to find any reasoning @noobtoob4lyfe ?

**Comment by noobtoob4lyfe** [comment]
@Keivin98  No, I have not received any response to this that would explain the error yet.

**Comment by vfan26** [comment]
same issue
